examples/oxr/python/test_oak_camera.py

```
@@ -66,62 +65,53 @@ def _run_schema_pusher(
+                    if (
+                        tracked.data is not None
+                        and tracked.data.sequence_number != last_seq.get(name, -1)
+                    ):
+                        metadata_samples[name] = metadata_samples.get(name, 0) + 1
+                        last_seq[name] = tracked.data.sequence_number
+
+                if int(elapsed) > last_print_time:
+                    last_print_time = int(elapsed)
+                    parts = []
+                    for name in stream_names:
+                        parts.append(f"{name}={metadata_samples.get(name, 0)}")
```

Ruff (0.15.6)

[warning] Use a list comprehension to create a transformed list

(PERF401)